local-setup/README.md

```diff
@@ -3,7 +3,11 @@
 ## Prerequisite
 - worker built with `make`
 - integritee-node built with `cargo build --release --features skip-ias-check`
-(in case you have a valid intel key, you can omit the `--features skip-ias-check`, but you must not use the subcommand `--skip-ra` in the config.json for the worker startup).
+
+In case you have
+- a sgx hardware and compile the worker with `SGX_MODE=HW` (default mode)
+- a valid intel IAS key (development key is fine)
+you can omit the `--features skip-ias-check` when building the node, but you must not use the subcommand flag `--skip-ra` in the config.json for the worker startup.
```

Comment:
can you shortly specify the config.json? where it is and what it is for?

Reply:
Good point. This is only introduced later. I'll clarify.

Reply:
Hope it's better understandable now, Including relative path linking 😊

Reply:
plain and clear. thank you!